docs/sections/guides.md

```diff
@@ -713,3 +647,459 @@ To re-run the entire workflow, use:
 ```sh
 popper run -f wf.yml
 ```
+
+
+## Computational research with R and RStudio Server
+
+This guide explains how to use Popper to develop and run reproducible workflows
+for computational research in fields such as bioinformatics, machine learning, physics 
+or statistics. 
+Computational research with R relies on complex software dependencies that are difficult to port 
+across environments. In addition, a typical workflow involves multiple dependent 
+steps which will be hard to replicate if not properly documented.
+Popper offers a solution to these challenges:
+- Poppers abstracts over software environments with [Linux containers](https://popper.readthedocs.io/en/latest/sections/concepts.html#glossary).
+- Poppers forces you to define your workflow explicetely such that it can be re-run in 
+in a single command.
+
+Popper thus provides an open-source alternative to managed solutions such as
+Code Ocean for reproducible computational research.
+
+### Pre-requisites
+
+You should have basic knowledge of:
```

Comment:
might look better in inline format "basic knowledge of Git, the command line, and R (...)